src/main/java/com/jcabi/github/mock/MkReleases.java

```diff
@@ -176,4 +205,37 @@ private String xpath() {
             this.coords
         );
     }
+
+    /**
+     * Check the first release is greater than last release.
+     * @param first First release.
+     * @param last Last release.
+     * @return True/False.
+     */
+    public boolean createLatest(
+            final Release first,
```

Comment:
@weisebrazil Plpease move the parameters to the first line, and wrap them only if they exceed 80 chars